snuba/datasets/dataset.py

```diff
@@ -153,12 +138,6 @@ def process_condition(self, condition) -> Tuple[str, str, Any]:
         """
         return condition
 
-    def get_split_query_spec(self) -> Union[None, ColumnSplitSpec]:
```

Comment:
This is another step in removing stuff from the dataset that should not be there.